fusesoc/edatools/xsim.py

```diff
@@ -55,19 +133,39 @@ def build_main(self):
         #Build simulation model
         args = []
         args += [ self.toplevel]
+        args += [ 'glbl' ]
```

Comment:
I don't think we should add glbl here, as that is only used for some Xilinx IP cores. Instead, we can set `toplevel = real_top_testbench glbl` in the .core file to indicate that we need two simulation toplevels

Reply:
I think it can confuse user. Not everyone is familiar with `glbl` and that it needs to be specified as second top for correct simulation. But I get the point of doing it another way. Then it has to be documented well.

Reply:
I think you're right. It probably doesn't hurt to always have this file when building with xsim or isim